### ISSUE TITLE
Analyze user feedback from engagement process

### ISSUE THREAD

**Issue by dawnbcgov** [bcgov/nr-epd-organics-info#96]
#### Description:

Analyze user feedback from engagement process and revisit features and scope of v1

#### Dependencies:

- Blocked by #91

**Comment by dawnbcgov** [comment]
Moving this to next sprint and making a new issue #143 for the feedback we did receive.
